doc/reference_cluster.xml

```diff
@@ -145,16 +145,80 @@ GROUP BY cid;
               <xref linkend="ST_ClusterKMeans"/>,
               <xref linkend="ST_ClusterIntersecting"/>,
               <xref linkend="ST_ClusterWithin"/>
+              <xref linkend="ST_ClusterIntersectingWin"/>
           </para>
 	  </refsection>
 
     </refentry>
 
+    <refentry id="ST_ClusterIntersectingWin">
+      <refnamediv>
+        <refname>ST_ClusterIntersectingWin</refname>
+
+        <refpurpose>Window function that returns a cluster id for each input geometry, clustering input geometries into connected sets.</refpurpose>
+      </refnamediv>
+
+      <refsynopsisdiv>
+        <funcsynopsis>
+          <funcprototype>
+            <funcdef>integer <function>ST_ClusterIntersectingWin</function></funcdef>
+            <paramdef><type>geometry</type> <parameter>g</parameter></paramdef>
```

Comment:
```suggestion
            <paramdef><type>geometry winset </type> <parameter>geom</parameter></paramdef>
```

I'd change to geom to be consistent with the other functions, unless you plan to at some point in time make this work for geography, might be a good time to experiment with anyelement :)

The winset is fluff to flag it as a window function so the garden test can test it and it also appears in the **15.2. PostGIS Window Functions** section.